### PR TITLE
ovirt_cluster: Fix documentation for gluster option

### PR DESCRIPTION
**Pull request by KKoukiou**
##### SUMMARY

Fix description of gluster option in ovirt_cluster module

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

ovirt_cluster.py

##### ANSIBLE VERSION

```
ansible 2.4
```

##### ADDITIONAL INFORMATION
```
More info about gluster option when configuring the cluster can be found here, https://www.ovirt.org/documentation/admin-guide/chap-Clusters/
```

### CODE REVIEW

File: lib/ansible/modules/cloud/ovirt/ovirt_cluster.py [hunk 1]
```diff
@@ -65,8 +65,9 @@
     gluster:
         description:
             - "If I(True), hosts in this cluster will be used as Gluster Storage
-               server nodes, and not for running virtual machines."
-            - "By default the cluster is created for virtual machine hosts."
+               server nodes."
+            - "By default the cluster is created for virtual machine hosts only."
+            - "Note that you cannot add oVirt Host (RHVH) to a Gluster-enabled cluster"
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_cluster.py]: I think upstream it's called `oVirt node`,  downstream RHV-H

Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_cluster.py]: I just copied the oVirt documentation from here, https://www.ovirt.org/documentation/admin-guide/chap-Clusters/ but you're probably right, I 'll repost.

File: lib/ansible/modules/cloud/ovirt/ovirt_cluster.py [hunk 2]
```diff
@@ -65,8 +65,9 @@
     gluster:
         description:
             - "If I(True), hosts in this cluster will be used as Gluster Storage
-               server nodes, and not for running virtual machines."
-            - "By default the cluster is created for virtual machine hosts."
+               server nodes."
+            - "By default the cluster is created for virtual machine hosts only."
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:69]: This is not entirely true, if you want to run VMs on a host you need to have `virt` enabled. But `gluster` is related to storage: if `gluster` is disabled, then you can use only classic storage like NFS, FC, iSCSI, but if `gluster` is enabled then you can also use Gluster storage.

File: lib/ansible/modules/cloud/ovirt/ovirt_cluster.py [hunk 3]
```diff
@@ -65,8 +65,9 @@
...
+            - "Note that you cannot add oVirt Node (RHV-H) to a Gluster-enabled cluster"
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:70]: Sandro, is above sentence correct? oVirt Node (NGN) can't be added to clusters with gluster feature enabled?

@sandrobonazzola

**Review by mwperina — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/ovirt/ovirt_cluster.py (hunk 3 above)
Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:70]: Please wait on this, it looks like this sentence that I found on docs, is not true. ""Note that you cannot add oVirt Node (RHV-H) to a Gluster-enabled cluster""

I 'll get back here when I get answer.

Comment (gundalow) [reply lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:70]: @KKoukiou Hi, just wondering if you got an answer to this.

Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:70]: @gundalow notreally AFAIK. But I 'll close the PR since it's stale long time.

**Pull request closed without merging**